package.json

```diff
@@ -198,6 +198,6 @@
   "dependencies": {
     "camelcase-keys-recursive": "^0.8.2",
     "reselect": "^2.5.4",
-    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#0e5db3eaa46d9a666ce96fe35ee7d6b71ff77b8b"
+    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#07e1e0f333491559a23c80074a44eee333d36b16"
```

Comment:
I made a PR, this will need to be updated again: https://github.com/department-of-veterans-affairs/vets-json-schema/pull/142